tests/prescribed_stokes_solution.prm

```diff
@@ -0,0 +1,181 @@
+# test PrescribedStokesSolution
```

Comment:
It would be nice to have a bit more detailed comment here about what the test does, like "This test is identical to xxx, except that we prescribe the velocity/pressure in the following way/using ..." and what the result should be.